provider/provider_test.go

```diff
@@ -13,10 +13,11 @@ import (
 
 	//"github.com/hashicorp/terraform-plugin-sdk/acctest"
 	//"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"math/rand"
```

Comment:
Not sure why this moved. `gofmt` is usually pretty aggro about include ordering.
Please verify that `gofmt -d .` doesn't flag anything.

Reply:
Oh sorry I didn't even notice this change, I ran `gofmt -d .` locally and nothing was outputted.